packages/radio/src/RadioCards/RadioCards.tsx

```diff
@@ -74,15 +74,25 @@ const ItemDescription = <T extends TgphElement>({
   size = "0",
   ...props
 }: ItemDescriptionProps<T>) => {
-  return <Button.Text as={"span"} size={size} {...props} />;
+  return (
+    <Button.Text
+      className="tgph-radio-card-description"
```

Comment:
Let's make sure we're passing `className` from props into all of these as well 

`className={clsx("tgph-radio-card-description", className)}`